### PR TITLE
DOMA-1213 add subscription settings screens

### PR DESCRIPTION
**Pull request by Dimitreee**

### CODE REVIEW

File: apps/condo/domains/common/components/CardContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const CardContainer = styled.div`
+  position: relative;
+  box-sizing: border-box;
+  border: 1px solid ${colors.lightGrey[5]};
+  border-radius: 8px;
+  padding: 29px 24px 80px 24px;
+  transition: ${transitions.elevateTransition};
+  cursor: pointer;
+  background-color: ${props => (props['data-status'] !== 'disabled' && props['data-status'] !== 'available')
```
Comment (AntonAL) [inline apps/condo/domains/common/components/CardContainer.tsx:12]:
```suggestion
  background-color: ${props => ['disabled', 'available'].includes(props['data-status'])
```

File: apps/condo/domains/subscription/components/SubscriptionPane.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                                                </Row>
+                                            </FocusContainer>
+                                        </Col>
+                                    )
+                                })}
+                            </Row>
+                        </Col>
+                    </Row>
+                </Col>
+                <Col span={24}>
```
Comment (AntonAL) [inline apps/condo/domains/subscription/components/SubscriptionPane.tsx:123]: I think, we should render this conditionally, if `activeSubscription` is present.

File: apps/condo/lang/ru.json [hunk 3]
```diff
@@ -562,6 +564,20 @@
...
+  "subscription.data.status": "Статус подписки",
+  "subscription.data.accountsCount": "Количество лицевых счетов",
+  "subscription.data.value": "Сумма",
+  "subscription.data.createBill": "Выставить счет",
+  "subscription.alert.subscription.payment.pending": "Запрос на выставление счета отправлен в отдел продаж, мы свяжемся с вами в ближайшее время",
+  "subscription.option.description.clients": "Для клиентов",
+  "subscription.option.description.account": "за лицевой счет",
+  "subscription.modal.complete.action": "Хорошо",
+  "subscription.modal.complete.description": "Мы свяжемся с вами в ближайшее время, для уточнения числа помещений и расчёта стоимости подписки.\nПредварительное количество помещений 25.",
+  "subscription.modal.complete.title": "Запрос на выставлени счёта принят",
```
Comment (AntonAL) [inline apps/condo/lang/ru.json:580]: Please, add English translations for these messages.

File: apps/condo/domains/common/components/CardContainer.tsx (hunk 1 above)
Comment (Dimitreee) [reply apps/condo/domains/common/components/CardContainer.tsx:12]: redurant file, removed

File: apps/condo/domains/subscription/components/SubscriptionPane.tsx (hunk 2 above)
Comment (AntonAL) [reply apps/condo/domains/subscription/components/SubscriptionPane.tsx:123]: I will do it after merge of `ServiceSubscription` schema.

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/subscription/components/SubscriptionPane.tsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+const StyledTitle = styled(Typography.Title)`
+  color: ${colors.sberPrimary[5]};
+`
+
+const getSubscriptionOptions = (intl, setIsVisible) => {
+    return [
+        {
+            renderTitle: () => (// Fetch from backend
+                <StyledTitle style={{ color: colors.sberPrimary[6] }}>
+                    1 ₽ / год
```
Comment (dkoviazin) [inline apps/condo/domains/subscription/components/SubscriptionPane.tsx:21]: Maybe, move to constants or translates?

File: apps/condo/lang/en.json [hunk 5]
```diff
@@ -565,6 +566,21 @@
...
+  "subscription.data.status": "Subscription status",
+  "subscription.data.accountsCount": "Number of personal accounts",
+  "subscription.data.value": "Total value",
+  "subscription.data.createBill": "Bill",
+  "subscription.alert.subscription.payment.pending": "An invoice request has been sent to the sales department, we will contact you shortly",
+  "subscription.option.description.clients": "For clients",
+  "subscription.option.description.account": "at personal expense",
+  "subscription.modal.complete.action": "OK",
+  "subscription.modal.complete.description": "We will contact you as soon as possible to clarify the number of premises and calculate the cost of the subscription.\nThe preliminary number of rooms is 25.",
+  "subscription.modal.complete.title": "Invoicing request accepted",
```
Comment (dkoviazin) [inline apps/condo/lang/en.json:583]: We need a tool to sync translates line numbers

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/lang/en.json (hunk 5 above)
Comment (AntonAL) [reply apps/condo/lang/en.json:583]: I have an idea, how to implement it.
DOMA-1272

Comment (Dimitreee) [reply apps/condo/lang/en.json:583]: We discuss this with @dkoviazin and @pahaz months ago;)

**Pull request merged**